src/app/not-found.tsx

```diff
@@ -0,0 +1,22 @@
+import Link from "next/link";
+
+export default function notfound() {
```

Comment:
One nit: any reason for not using Pascal case?
i.e:
```suggestion
export default function NotFound() {
```

Reply:
I'll make necessary changes